### PR TITLE
Manual placement mode

### PR DESCRIPTION
**Pull request by JakubPlatzek**

### CODE REVIEW

File: Assets/ManualPlacement.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                    BlockPress();
+                }
+            }
+        }
+    }
+
+    void BlockPress(){
+        pressed = false;
+        canPress = false;
+        Invoke("UnblockPress", 1f);
```
Comment (tomondre) [inline Assets/ManualPlacement.cs:47]: Whats the point of blocking the press and then unblocking it in one second? Is it for testing purposes?

Comment (JakubPlatzek) [reply Assets/ManualPlacement.cs:47]: No, otherwise it spams the button.

Comment (JakubPlatzek) [reply Assets/ManualPlacement.cs:47]: Might have to do different handling for the future.

Comment (tomondre) [reply Assets/ManualPlacement.cs:47]: Isnt it better to change the listener to something like `onPressDown` or `onPressDown`? E.g. on a higher level component that is actually calling this function?

**Pull request merged**